app/src/main/java/de/stefanmedack/ccctv/ui/events/EventsActivity.kt

```diff
@@ -35,11 +41,19 @@ class EventsActivity : BaseInjectableActivity() {
         fun start(activity: Activity, conference: Conference) {
             val intent = Intent(activity.baseContext, EventsActivity::class.java)
             intent.putExtra(CONFERENCE_ID, conference.id)
-            intent.putExtra(CONFERENCE_TITLE, conference.title)
+            intent.putExtra(EVENTS_VIEW_TITLE, conference.title)
             intent.putExtra(CONFERENCE_LOGO_URL, conference.logoUrl)
 
             activity.startActivity(intent, ActivityOptionsCompat.makeSceneTransitionAnimation(activity).toBundle())
         }
+
+        fun start(activity: Activity, searchQuery: String) {
+            val intent = Intent(activity.baseContext, EventsActivity::class.java)
+            intent.putExtra(EVENTS_VIEW_TITLE, activity.getString(R.string.events_view_search_result_header).format(searchQuery))
```

Comment:
A better way to formant Strings is to use the replace possibilities by `getString()`. You can find more information and a googd example on how to use this feature [here under Formatting Strings](https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling)